android/jni/app/organicmaps/Map.cpp

```diff
@@ -22,147 +22,159 @@ void OnRenderingInitializationFinished(std::shared_ptr<jobject> const & listener
 
 extern "C"
 {
-using namespace storage;
+JNIEXPORT jboolean JNICALL
+Java_app_organicmaps_Map_nativeCreateEngine(JNIEnv * env, jclass clazz,
+                                                        jobject surface, jint density,
```

Comment:
1. It doesn't look aligned.
2. Please avoid declaring unused function parameters like clazz, or comment them out.